components/Footer.js

```diff
@@ -0,0 +1,20 @@
+// import React from 'react'
```

Comment:
remove this file if you're not using it or make a comment on why you're keeping it